src/unix/proctitle.c

```diff
@@ -50,7 +50,9 @@ char** uv_setup_args(int argc, char** argv) {
 
   process_title.str = argv[0];
   process_title.len = argv[argc - 1] + strlen(argv[argc - 1]) - argv[0];
+#ifndef __MVS__
   assert(process_title.len + 1 == size);  /* argv memory should be adjacent. */
```

Comment:
In fact, I am not so sure that it is safe to overwrite arg[0] with uv_set_proctitle
So, I will disable uv_set/get_proctitle on zOS just as it is on aix